code/spring-boot-telemetry/src/main/resources/application.properties

```diff
@@ -1,7 +1,6 @@
 spring.application.name=SpringVeggieService
 spring.datasource.name=veggies
-spring.datasource.url=jdbc:otel:postgresql://db:5432/demo
-spring.datasource.driver-class-name=io.opentelemetry.instrumentation.jdbc.OpenTelemetryDriver
```

Comment:
will the class name not be needed anylonger?

Reply:
@SandraAhlgrimm  This new version provides out of the box instrumentation for JDBC (without any configuration, without updating the JDBC URL and JDBC driver class).